src/thematic/4_vegetation/sfp-401_vegopname-terrest_nl/07_stappenplan.Rmd

```diff
@@ -61,7 +61,7 @@ deze worden samengevoegd met per soort het maximum van beide ingeschatte
 bedekkingen.
 
 
-```{r image1, fig.cap = 'Monitoringsperiode naargelang biotoopgroep (Vriens *et al.*, 2011)', out.width= '100%' }
+```{r image1, fig.cap = 'Monitoringsperiode naargelang biotoopgroep [@RN158]', out.width= '100%' }
```

Comment:
```suggestion

(ref:image1) Monitoringsperiode naargelang biotoopgroep [@RN158]

```{r image1, fig.cap = '(ref:image1)', out.width= '100%' }
```

In case figure captions contain special characters or bib references, you need to work with a text reference. This is explained [here](https://bookdown.org/yihui/bookdown/markdown-extensions-by-bookdown.html#text-references)